FirebaseInAppMessaging/CHANGELOG.md

```diff
@@ -1,114 +1,114 @@
-# 2022-01 -- v8.12.0
+# v8.12.0
 - [fixed] In-App Messaging's test message does not include appData in response. This SDK fix will work once the backend is also updated (#9126).
 
-# 2021-12 -- v8.11.0
+# v8.11.0
 - [fixed] InApp message is shown every new session (#8907).
 - [fixed] Duplicate messages can occur when two campaigns are triggered by different events in In-App Messaging (#9070).
 
-# 2021-8 -- v8.6.0
+# v8.6.0
 - [changed] Replaced conditionally-compiled APIs with `API_UNAVAILABLE` annotations on unsupported platforms (#8480).
 
-# 2021-8 -- v.8.5.0
+# v.8.5.0
 - [added] Added support for unit testing with in-app message data objects (#8351).
 - [added] Added support for prototyping custom in-app message views in SwiftUI (#8351).
 
-# 2021-7 --- v8.4.0
+# 7 --- v8.4.0
```

Comment:
7 --- should be removed

Reply:
Done here and below in [f16b82e](https://github.com/firebase/firebase-ios-sdk/pull/9777/commits/f16b82eafaa5bc7323b37ff85727173e980f76c6)